src/actions/compass.ts

```diff
@@ -0,0 +1,45 @@
+import {SpecQueryModelGroup} from 'compassql/build/src/model';
+import {Query} from 'compassql/build/src/query/query';
+import {Schema} from 'compassql/build/src/schema';
+import {Dispatch} from 'redux';
+import {ThunkAction} from 'redux-thunk';
+import {ReduxAction} from './redux-action';
+
+import {fetchCompassRecommends} from '../api/api';
+import {State} from '../models/index';
+import {getQuery, getSchema} from '../selectors';
+import {Action} from './index';
+
+export type CompassAction = CompassRecommendsRequest | CompassRecommendsRecieve;
```

Comment:
So the data store is going to be renamed `result` 

should these actions follow that naming convention and be called `ResultAction` ? instead of `Query` ?